src/CSSList/CSSList.php

```diff
@@ -296,6 +296,22 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Insert an item before its sibling.
+     *
+     * @param RuleSet|CSSList|Import|Charset $oItem The item.
+     * @param RuleSet|CSSList|Import|Charset $oSibling The sibling.
+     */
+    public function insertBefore($oItem, $oSibling): void
+    {
+        $iIndex = in_array($oSibling, $this->aContents);
```

Comment:
Please always use the strict version of `in_array`:

```suggestion
        $iIndex = in_array($oSibling, $this->aContents, true);
```

Reply:
Shouldn't we add this rule then? https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/strict/strict_param.rst

Reply:
Yes, definitely: #584